system-addon/content-src/components/TopSites/TopSitesEdit.jsx

```diff
@@ -121,8 +133,8 @@ export class _TopSitesEdit extends React.PureComponent {
           <div className="modal-overlay" onClick={this.onModalOverlayClick} />
           <div className="modal">
             <TopSiteForm
-              label={this.props.TopSites.rows[editIndex].label || this.props.TopSites.rows[editIndex].hostname}
```

Comment:
Do we not want `.hostname` anymore? It might have been from old pinned tiles?

Reply:
oh. copy paste fail there. let me test something